build.sbt

```diff
@@ -345,7 +344,7 @@ lazy val dockerSettings = Def.settings(
       s"tar -xf $sbtTgz",
       s"rm -f $sbtTgz"
     ).mkString(" && ")
-    val millVer = Dependencies.millVersion
+    val millVer = Dependencies.millScriptVersion
     val millBin = s"$binDir/mill"
     val installMill = Seq(
       s"$curl $millBin https://github.com/lihaoyi/mill/releases/download/${millVer.split("-").head}/$millVer",
```

Comment:
you'll want to update that part, because `https://github.com/lihaoyi/mill/releases/download/0.11.0/0.11.0-M11` is wrong